src/components/Home/StationModalContent.tsx

```diff
@@ -50,39 +58,167 @@ export default function StationModalContent({
         isOpen && getWeatherData();
     }, [isOpen]);
 
+    const transformDateToReadable = (inputDate: Date): string => {
+        const options: Intl.DateTimeFormatOptions = {
+            weekday: "short",
+            month: "short",
+            day: "numeric",
+            hour: "numeric",
+            minute: "numeric",
+            hour12: false,
+        };
+        const date: Date = new Date(inputDate);
+        const formattedDate: string = date.toLocaleDateString("en-US", options);
+        return formattedDate;
+    };
+    /**
+    * @todo Propably move these literals to some place better. Maybe i18n?
```

Comment:
Nice idea, would you like to create a ticket for that?
In the meanwhile, do you think it's worth defining it as a const at the top?

Reply:
Yeah, ticket created. https://tzoupy.atlassian.net/browse/FWM-39
I am going to move it to the top, makes more sense in case we need it for something else.